docs/content/sre/tenant-operator/features.md

```diff
@@ -53,7 +53,7 @@ Tenant-operator can manage Mattermost to create Teams for tenant users. All tena
 
 ## Everything as Code/GitOps Ready
 
-Tenant-Operator is designed and built to be 100% OpenShift native and to be configured and managed the same familiar way as native OpenShift resources so is perfect for modern shops that are dedicated to GitOps as it is fully configurable using Custom Resources 
+Tenant-Operator is designed and built to be 100% OpenShift native and to be configured and managed the same familiar way as native OpenShift resources so is perfect for modern shops that are dedicated to GitOps as it is fully configurable using Custom Resources
```

Comment:
`OpenShift-native` instead of `OpenShift native` maybe?;
Please add a full stop at the end